### DISCUSSION TITLE
thanks..

### DISCUSSION THREAD

**Discussion by sachinchaudhary1808** [MikaelFangel/nixvim-config#57] [General]
i liked your config.. i mean that git stuffs, also that thing were i can see the nesting stuff.. also startup time... thanks...

**Comment by MikaelFangel** [comment]
Enjoy and please let me know if you have any suggestions or experience issues 😇

**Reply by sachinchaudhary1808** [reply]
keymapping indication.... 
the big issue i see is like about nixvim... is if someone like me adopts nixvim without knowing neovim.. and lua and all.. he needs some good docs.. like add more comments that explaing each or some things that a user like me need to understand... i hope u understand me...

**Reply by sachinchaudhary1808** [reply]
i know nix is for devs.. but i wanna use it, guys like u can make it more less painful...

**Reply by MikaelFangel** [reply]
I see. So updating the wiki with keymaps would maybe be an okay solution?

**Reply by sachinchaudhary1808** [reply]
so may be u can use which key for leader keys....(which key plugin...)
add comments in file like which thing or plugin what it does.. if u don't wanna make your code bloted then do it in like last lines explaning things or do it on wiki......

**Reply by sachinchaudhary1808** [reply]
the plugin thing (what it does) is not required for experienced or who has used neovim but for newbie it's better....

**Reply by MikaelFangel** [reply]
Okay thanks for the feedback I'll give it some consideration 😊

**Reply by MikaelFangel** [reply]
As a beginning on resolving this I've added the which-key plugin in #58 so when you pres `<leader>` (spacebar) you get what options that are available.

**Reply by MikaelFangel** [reply]
And now a short wiki and an updated readme:
https://github.com/MikaelFangel/nixvim-config/wiki/Key-Mappings

**Reply by sachinchaudhary1808** [reply]
i see sir, u have done very good work.. thanks it will help a beginner...
